myhpi/static/css/myHPI.css

```diff
@@ -0,0 +1,217 @@
+/**
+Notice the myHPI.scss in static/scss/myHPI.scss
+**/
+
+body {
+    padding-top: 1rem;
+}
+
+a {
+    text-decoration: none;
+}
+
+.container-lg.page {
+    padding: 0 !important;
+}
+
+.page {
+    margin: auto;
+}
+
+.page-content {
+    width: 1000px;
+    margin: auto;
+}
+
+.navbar-myhpi {
+    position: sticky;
+    z-index: 999;
+    top: 0;
+    display: flex;
+    flex-direction: column;
+}
+
+.navbar-myhpi::before {
+    display: inline-block;
+    content: '';
+    height: 0.5rem;
+    border-top-left-radius: var(--myhpi-round-border);
+    border-top-right-radius: var(--myhpi-round-border);
+}
+
+.myhpi-accent-yellow .navbar-myhpi::before,
+.myhpi-accent-yellow .bg-accent {
+    background-color: var(--bs-secondary);
+}
+
+.myhpi-accent-yellow .border-accent {
+    border-color: var(--bs-secondary);
+}
+
+.myhpi-accent-yellow .breadcrumb .active {
+    color: var(--bs-secondary);
+}
+
+.navbar-top,
+.navbar-bottom {
+    display: flex;
+}
+
+.navbar-top {
+    border-bottom: solid 2px var(--bs-border);
+    z-index: 1;
+}
+
+.navbar-bottom {
+    overflow: hidden;
+    background-color: var(--bs-body-bg-light);
+}
+
+.navbar-bottom-content .nav-item-container {
+    margin-bottom: 3px;
+}
+
+.navbar-bottom-line {
+    position: absolute;
+    width: 100%;
+    border-bottom: solid 3px var(--bs-border);
+    bottom: 0px;
+}
+
+.nav-brand {
+    display: inline-block;
+    width: 4rem;
+    margin: 1rem;
+    margin-left: 0;
+}
+
+.nav-level-0 {
+    grid-area: menu-level-0;
+    background-color: var(--bs-white);
+    display: flex;
+    align-items: center;
+}
+
+.nav-level-1 {
+    /* equals .nav-brand width + margin,
+       minus the .nav-item border-width */
+    margin-left: calc(5rem - 3px);
+}
+
+.nav-level-1,
+.nav-level-2 {
+    width: 20rem;
+}
+
+.nav-item-container {
+    display: flex;
+}
+
+.nav-level-1 .nav-item-container,
+.nav-level-2 .nav-item-container {
+    flex-direction: column;
+}
+
+nav .nav-link,
+nav .nav-link:focus {
+    color: var(--bs-gray-700);
+}
+
+nav .nav-link:hover,
+nav .nav-link:focus-visible,
+nav .nav-link[aria-expanded=true] {
+    color: var(--bs-dark);
+}
+
+.page:not(.navbar-myhpi) .page-content {
+    padding-top: 1.5rem;
+    padding-bottom: 1.5rem;
+}
+
+.page-title .underline {
+    border-bottom: solid 5px var(--bs-secondary);
+}
+
+.page-title {
+    display: inline-block;
+    font-weight: 600;
+    margin-bottom: 2.3rem;
+    /*   color: #FFF;
+    background-color: var(--bs-primary);
+    padding: 0.8rem 1.5rem;
+    padding-left: 10rem;
+    margin-left: -10rem;
+    margin-bottom: 1rem;
+    border-top-right-radius: 5px;
+    border-bottom-right-radius: 5px; */

```

Comment:
just from reading the code: what is that for?

Reply:
Ohh, that's an alternative design for page titles which i forgot to delete. Going to do this now.
Looked something like this:
![grafik](https://user-images.githubusercontent.com/44174681/168327024-c68f78a7-fea7-424c-b0ce-5c7245a5b6c4.png)